### PR TITLE
Added unlc concordance values

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/641.

For any feature that current has a `unlc:subdivision` property, a `unlc:id` concordance value was appended to the record's `wof:concordances` list.

As commented in the issue, the original `unlc:subdivision` property was kept in each record.

### CODE REVIEW

**Comment by stepps00** [comment]
Fix `data/136/251/273/136251273.geojson` before merge (`misc:*` props).

File: data/136/251/273/136251273.geojson [hunk 1]
```diff
@@ -13,6 +13,25 @@
     "iso:subdivision":"CA-QC",
     "lbl:latitude":52.03334,
     "lbl:longitude":-72.765013,
+    "misc:qs_a0":"Canada",
```
Comment (nvkelso) [inline data/136/251/273/136251273.geojson]: This should be `qs:a0` not `misc:qs_a0`.

**Comment by nvkelso** [comment]
@stepps00 why `"unlc:id"` instead of `"unlc:subdivision"`?

```
    "unlc:subdivision":"AD-04",
     "wk:wordcount":993,
     "wof:belongsto":[
         102191581,
         85632343
     ],
     "wof:breaches":[],	
     "wof:concordances":{	
          "gn:id":3040131,	
          "gp:id":20070555,	
          "qs:id":1102991,	
          "unlc:id":"AD-04",
          "wd:id":"Q24276",	
          "wk:page":"La Massana"
      },
```

Other than that I spot checked these and they look good (except https://github.com/whosonfirst-data/whosonfirst-data/pull/822#discussion_r121244445).

File: data/136/251/273/136251273.geojson (hunk 1 above)
Comment (stepps00) [inline data/136/251/273/136251273.geojson]: Yeah, that's a bug in my callback script. It only affects the `136251273.geojson ` record - I've noted to fix it in an upcoing commit.

**Comment by stepps00** [comment]
@nvkelso - I wanted the concordance to be standardized like other concordance values, though I have found `fips:code` in the concordances list. Would you expect this to be a `unlc:subdivision` concordance instead of `unlc:id`?

**Comment by nvkelso** [comment]
@stepps00 Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/822#issuecomment-307531059, I'd expect it to match the `id` value given in the source JSON here: https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/unlocode.json#L6. In this case they _do_ match – but it's not consistent with `unlc:subdivision` that we use elsewhere. 

Looking at the upstream source metadata at http://www.unece.org/cefact/codesfortrade/codes_index.html it looks like all follow the `CC-az1` regardless of subdivision (WOF: `region`) or port (WOF: `venue` or `campus`) or other feature class so I think using `id` here is okay, but confusing since the source doesn't concatenate them together (instead they'd have `CC` in one column and `az1` in the next. Deserves a note in our https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/unlocode.json as to how we construct the `id`. Suggest adding something like:

```
WOF concatenate the UN/LOCODE 2 character country code and 3 character
feature code with a `-` deliminator and name the result the `id`.
```

**Comment by stepps00** [comment]
I've updated the `UNLOCODE` source in this commit: https://github.com/whosonfirst/whosonfirst-sources/commit/e37ca85dba0c3e5983cfa98a5cf21ce269e15c83.

I will push a new commit to this branch to change the existing `unlc:subdivision` concordance to `unlc:id` shortly.

**Comment by stepps00** [comment]
@nvkelso - New `unlc:id` property should be included in all records, as well as `qs:` fixes for the record of Quebec, and a note in the UNLOCODE source file.

Ready for another review.

**Comment by stepps00** [comment]
The two most recent commits take care of any merge conflicts in this branch.

This PR is a good candidate to test the pipeline tools; no PIP work is required for this PR.

**Comment by nvkelso** [comment]
Or rather @stepps00 wants to merge it with @dphiffer using new tool @dphiffer built.

**Comment by stepps00** [comment]
Closing in favor of #882.

**Pull request closed without merging**